src/app/event/[id]/page.tsx

```diff
@@ -1,13 +1,141 @@
+                <h2 className="mt-4 text-center text-xl font-medium leading-normal md:px-0 lg:mt-0 lg:whitespace-nowrap lg:text-start lg:text-3xl">
+                  Sportuj s Instruktorkou{' '}
+                  <Link href="/" className="hover:text-primary">
+                    {mockEventHost.name}
+                  </Link>
+                </h2>
+                <p className="font-light lg:whitespace-nowrap">
+                  Instruktorkou na SportujSpolu od roku {mockEventHost.since}
+                </p>
+              </div>
+            </div>
+            <p
+              className="mt-8 text-lg font-light"
+              dangerouslySetInnerHTML={{ __html: mockEventHost.description }}
+            />
+            <hr className="my-16 w-full border-t border-light-gray" />
+          </section>
+          <section className="w-full">
+            <h2 className="flex flex-col px-20 text-center text-xl font-medium leading-normal md:flex-row md:px-0 lg:text-start lg:text-3xl">
+              <StarRating
+                className="h-6 w-6 whitespace-nowrap px-4"
+                rating={mockEventHost.rating}
+              />
+              <span>
+                {mockEventHost.rating} ({mockEventHost.reviewsCount} hodnocení)
+              </span>
+            </h2>
+            {mockEventHost.reviews.map((review, hostId) => (
+              <div key={hostId}>
+                <div key={review.id} className="relative mt-10 flex flex-row">
+                  <Link href="/" className="relative mr-4 h-12 w-12">
+                    <Image
+                      alt=""
+                      src={review.image}
+                      className="rounded-full object-cover"
+                      fill
+                    />
+                  </Link>
+                  <div className="mr-2">
+                    <h3 className="items-center text-base md:text-lg">
+                      <span className="text-center font-medium leading-normal lg:text-start">
+                        {review.name}{' '}
+                      </span>
+                      <span className="font-light">
+                        hodnotí akci <StarRating rating={review.rating} />
+                      </span>
+                    </h3>
+                    <p className="font-light">{review.date}</p>
+                  </div>
+                </div>
+                <p
+                  className="ml-16 mt-4 line-clamp-6 text-lg font-light"
+                  dangerouslySetInnerHTML={{ __html: review.comment }}
+                />
+              </div>
+            ))}
+          </section>
+        </div>
+        <EventDetail className="sticky hidden lg:block" event={event} />
+      </div>
+      <hr className="my-16 border-t border-light-gray" />
+      <section>
+        <h2 className="px-20 text-center text-xl font-medium leading-normal md:px-0 lg:text-start lg:text-3xl">
+          Další sportovní akce
+        </h2>
+        <Events events={events} />
```

Comment:
Do some slicing, we dont need to see all. It would be probably "Podobne sportovni akce" the moment we have some filters in place